scripts/check-dependencies.js

```diff
@@ -51,24 +56,29 @@ const excludedPackages = [
 
 // We are going to copy these files into each package from this script, in order to
 // standardize them.
-const commonConfigs = [
-    '.prettierrc.yaml'
-]
+const commonConfigs = ['.prettierrc.yaml']
+
+// For dependencies that we want to use the same version in all packages,
+// use this package as the source of truth for the version to use
+const COMMON_DEPS_SOURCE_OF_TRUTH = 'template-retail-react-app'
```

Comment:
Turns out `internal-lib-build` doesnt' actually use `cross-env`, so I changed the source of truth to `template-retail-react-app`.